src/routes/routes.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express'
+import { HomeController } from '../controllers/HomeController';
+
+
+const router: Router = Router();
+
+
+/**
+ * Controllers (route handlers).
+ */
+const homeController = new HomeController();
+
+router.get("/", homeController.sayHello);
+
+export const appRouter = router;
```

Comment:
I think `appRouter` alias is not necessary here and it just make it more difficult to read. Could you just export router without having an alias variable?